### ISSUE TITLE
Colon rule not autocorrecting dictionary

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#2049]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? 0.25.0
* Installation method used (Homebrew, CocoaPods, building from source, etc)? Homebrew
* Paste your configuration file:

No configuration needed.

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
  If so, paste their relative paths and respective contents.

No.

* Which Xcode version are you using (check `xcode-select -p`)?

9.2.

* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
  to quickly test if your example is really demonstrating the issue. If your example is more
  complex, you can use `swiftlint lint --path [file here] --no-cache --enable-all-rules`.

```swift
// this should autocorrect
dict = [Keys.CELL_TYPE : SNCFObotoCellType.radio]
```

**Comment by marcelofabri** [comment]
This actually works.
